src/reducers/versions.spec.tsx

```diff
@@ -1774,4 +1778,203 @@ describe(__filename, () => {
       expect(state.versionInfo[beginFetchVersionId]).toEqual(undefined);
     });
   });
+
+  type TestHunkChange = {
+    lineNumber: number;
+    type: string;
+  };
+
+  type TestHunk = TestHunkChange[];
+
+  // This helper accepts an array of arrays, each of which is a list of changes
+  // which will be contained in a hunk.
+  const createDiff = (testHunks: TestHunk[]): DiffInfo => {
+    const diffSample = parseDiff(diffWithDeletions)[0];
+
+    const hunks = testHunks.map((hunk) => {
+      return {
+        changes: hunk.map((change) => {
+          return {
+            content: 'the content of the change is irrelevant to the tests',
+            isDelete: change.type === 'delete',
+            isInsert: change.type === 'insert',
+            isNormal: change.type === 'normal',
+            lineNumber: change.lineNumber,
+            oldLineNumber:
+              change.type === 'normal' ? change.lineNumber : undefined,
+            type: change.type as ChangeType,
```

Comment:
You don't have to make this type assertion if you reference the real property in `TestHunkChange`. I think that would also be a good idea to keep the two types in sync. Example:

```diff
diff --git a/src/@types/react-diff-view/index.d.ts b/src/@types/react-diff-view/index.d.ts
index 0f07a6d..7a2e56a 100644
--- a/src/@types/react-diff-view/index.d.ts
+++ b/src/@types/react-diff-view/index.d.ts
@@ -3,7 +3,7 @@
 declare module 'react-diff-view' {
   type ChangeType = 'delete' | 'insert' | 'normal';
 
-  type ChangeInfo = {
+  export type ChangeInfo = {
     content: string;
     isDelete?: boolean;
     isInsert?: boolean;
diff --git a/src/reducers/versions.spec.tsx b/src/reducers/versions.spec.tsx
index 89d7176..097392d 100644
--- a/src/reducers/versions.spec.tsx
+++ b/src/reducers/versions.spec.tsx
@@ -1,5 +1,5 @@
 import { push } from 'connected-react-router';
-import { ChangeType, parseDiff } from 'react-diff-view';
+import { ChangeInfo, ChangeType, DiffInfo, parseDiff } from 'react-diff-view';
 import { getType } from 'typesafe-actions';
 
 import { actions as errorsActions } from './errors';
@@ -1780,15 +1780,13 @@ describe(__filename, () => {
   });
 
   type TestHunkChange = {
-    lineNumber: number;
-    type: string;
+    lineNumber: ChangeInfo['lineNumber'];
+    type: ChangeInfo['type'];
   };
 
-  type TestHunk = TestHunkChange[];
-
   // This helper accepts an array of arrays, each of which is a list of changes
   // which will be contained in a hunk.
-  const createDiff = (testHunks: TestHunk[]): DiffInfo => {
+  const createDiff = (testHunks: TestHunkChange[][]): DiffInfo => {
     const diffSample = parseDiff(diffWithDeletions)[0];
 
     const hunks = testHunks.map((hunk) => {
@@ -1802,7 +1800,7 @@ describe(__filename, () => {
             lineNumber: change.lineNumber,
             oldLineNumber:
               change.type === 'normal' ? change.lineNumber : undefined,
-            type: change.type as ChangeType,
+            type: change.type,
           };
         }),
         content: 'the content of hunks is irrelevant to the tests',
```